### ISSUE TITLE
RFE: Consider binding Esc to quit by default

### ISSUE THREAD

**Issue by contyk** [patroclos/PAmix#66]
Most similar apps can be closed by hitting the Escape key while pamix only responds to `q` by default.

I've tried to configure this in `pamix.conf` but couldn't figure out how to map that specific key to this action. Regardless, I think Escape should be supported by default.

**Comment by patroclos** [comment]
Regarding configuration, pamix uses [keyname](https://man7.org/linux/man-pages/man3/keyname.3x.html), which supports caret notation.

So binding ESC to quit would be `bind ^[ quit`

**Comment by contyk** [comment]
Yes, I looked at that and had no luck with it. Also blindly tried random key names, with and without the `KEY_` prefix, different letter cases. Nothing worked, sadly.
